DEV_GUIDE.md

```diff
@@ -53,5 +55,48 @@ Run the following command to format the source code and organize the imports as
 $ mvn process-sources
 ```
 
+## Running Integration Tests on Podman
 
+There is an incompatibility between HTTP connection timeout expectations of 
+[testcontainers-java](https://github.com/testcontainers/testcontainers-java) and the Podman API. This
+can result in sporadic test failures when running the Integration Tests under Podman.  It manifests as
+failed or hanging REST API calls that lead to test failures and test hangs.
+
+It affects Linux and Mac OS X.
+
+To workaround around the issue, turn off the `service_timeout` by following these instructions.
+
+### MacOS X
+
+Start the `podman` machine as normal, then:
+
+```shell
+echo 'mkdir -p /etc/containers/containers.conf.d && printf "[engine]\nservice_timeout=0\n" > /etc/containers/containers.conf.d/service-timeout.conf' |  podman machine ssh --username root --
+```
+
+### Linux
+
+As a privileged user:
+
+```shell
+mkdir -p /etc/containers/containers.conf.d && printf "[engine]\nservice_timeout=0\n" > /etc/containers/containers.conf.d/service-timeout.conf
```

Comment:
Not verified.

Reply:
On linux it might be better going into the user's home dir:
`mkdir -p  $HOME/.config/containers/ && printf "[engine]\nservice_timeout=0\n" > $HOME/.config/containers/containers.conf`

Reply:
confirming the /etc/ style works, let's go with that :+1:

Reply:
`/etc` assumes rootful AFICT the `$HOME` version is used for rootless podman setups.